src/widgets/buttons.ts

```diff
@@ -394,7 +404,7 @@ export function setImage (element: HTMLElement, thing: NamedNode) { // 20191230a
 // See eg http://stackoverflow.com/questions/980855/inputting-a-default-image
 function faviconOrDefault (dom: HTMLDocument, x: NamedNode) {
   var image = dom.createElement('img')
-  ;(image as any).style = style.iconStyle
+    ; (image as any).style = style.iconStyle
```

Comment:
what happened to the indent here, did your editor do this?

Reply:
yeah, it's my save.  I've tried to change it, but it just doesn't seem to work and the linter doesn't fix it.  is this okay or a problem?

Reply:
sure, no problem